dotcom-rendering/src/web/components/Picture.tsx

```diff
@@ -16,12 +16,13 @@ type Props = {
 
 type ResolutionType = 'hdpi' | 'mdpi';
 
-const getClosestSetForWidth = (
+export const getClosestSetForWidth = (
 	desiredWidth: number,
 	inlineSrcSets: SrcSetItem[],
 ): SrcSetItem => {
 	// For a desired width, find the SrcSetItem which is the closest match
-	const sorted = inlineSrcSets.sort((a, b) => b.width - a.width);
+	// A match greated than the desired width will always be picked when one is available
+	const sorted = inlineSrcSets.slice().sort((a, b) => b.width - a.width);
```

Comment:
Adding `.slice()` to avoid sideeffects as `.sort` modifies the original array

Reply:
such awareness of the evil of mutation...